src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -94,6 +94,7 @@
                     + "                                         ERROR,WARN,INFO,DEBUG,TRACE [default: INFO].\n"
                     + " -Q, --quickcheck-test-mode              Run in QuickCheck test mode\n"
                     + " -P <seconds>, --cm-poll-interval        Configuration manager poll interval [default: 1]\n"
+                    + " -v, --verify                            Compute and verify checksums for log entries.\n"
```

Comment:
my vote checksum mechanism should be there. Here are the scenarios we consider:
1. customer starts with no checksum and turns on checksum later ? We should be able to deal with entries that do not have checksum.
2. customer starts with checksum and (though unlikely for such a customer) then switches off checksum. This should be easy.
3. Server/component restarts should not flip the checksum behavior specified by the customer.